pkgs/by-name/ma/mapproxy/package.nix

```diff
@@ -2,24 +2,27 @@
   lib,
   fetchFromGitHub,
 
-  python3Packages,
+  python312Packages,
 }:
 
+let
+  python3Packages = python312Packages;
```

Comment:
Why is Python version pinned to 3.12 ? I can't see anything related mentioned in mapproxy 5.0.0. release notes .